src/main/java/commands/ByeCommand.java

```diff
@@ -0,0 +1,30 @@
+package commands;
+
+import common.TaskList;
+import storage.Storage;
+import ui.Ui;
+
+public class ByeCommand extends Command {
+
+    /**
+     * The user interface will show an ending message when this method is called.
+     *
+     * @param list List which stores all Tasks in the chatbot.
+     * @param ui The interface which the user will be interacting with.
+     * @param storage Stores previous and current Task objects.
+     */
+    @Override
+    public void execute(TaskList list, Ui ui, Storage storage) {
+        ui.showEnd();
```

Comment:
I may have missed it, but where is the part where the tasks get stored from the tasklist to the text file?